### DISCUSSION TITLE
Add the contain CSS property

### DISCUSSION THREAD

**Discussion by stijns96** [tailwindlabs/tailwindcss#12900] [Ideas]
Hi,

I would like to see the contain [CSS](https://developer.mozilla.org/en-US/docs/Web/CSS) property in Tailwind. Especially the 'contain:paint' could be very useful.

For me it's kind of a no-brainer.

**Comment by PrivFelixKniest** [comment]
Yeah I'm looking for it too right now, I need to add position sticky with overflow hidden, so contain: paint is what I need, I think.

**Comment by silverwind** [comment]
At least `contain: paint` and `contain: inline-size` are very useful properties.

**Comment by safwansamsudeen** [comment]
How has this not been done yet?
